### PR TITLE
CORDA -4030 - add flow to request membership attribute changes

### PR DESCRIPTION
**Pull request by nikinagy**
Additional PR for : https://r3-cev.atlassian.net/browse/CORDA-4030

1) Added additional checks for participants
2) Removed unnecessary schema class
3) Moved audit logs to the start of the flow

### CODE REVIEW

File: business-networks-workflows/src/main/kotlin/net/corda/bn/flows/BNService.kt [hunk 1]
```diff
@@ -117,6 +117,7 @@ class BNService(private val serviceHub: AppServiceHub) : SingletonSerializeAsTok
         }
     }
 
+
```
Comment (tubibuto) [inline business-networks-workflows/src/main/kotlin/net/corda/bn/flows/BNService.kt]: Extra new line

**Review by tubibuto — DISMISSED** [review]
Looks good, one tiny change and good to go

File: business-networks-workflows/src/main/kotlin/net/corda/bn/flows/BNService.kt (hunk 1 above)
Comment (nikinagy) [reply business-networks-workflows/src/main/kotlin/net/corda/bn/flows/BNService.kt]: removed

**Review by tubibuto — APPROVED** [review]

**Review by bpaunescu — APPROVED** [review]
OK

**Pull request merged**